src/platform/unix.rs

```diff
@@ -59,7 +62,15 @@ impl<T> EventLoopBuilderExtUnix for EventLoopBuilder<T> {
 }
 
 /// Additional methods on `Window` that are specific to Unix.
-pub trait WindowExtUnix {
+pub trait WindowExtUnix<T> {
+  /// Create a new Tao window from an existing GTK window. Generally you should use
+  /// the non-Linux `WindowBuilder`, this is for those who need lower level window access
+  /// and know what they're doing.
+  fn new_from_gtk_window(
+    event_loop_window_target: &UnixEventLoopWindowTarget<T>,
+    window: gtk::ApplicationWindow,
+  ) -> UnixWindow;
```

Comment:
```suggestion
  fn new_from_gtk_window<T: 'static>(
    event_loop: &EventLoopWindowTarget<T>,
    window: gtk::ApplicationWindow,
  ) -> Self;
```

Reply:
Sorry, forgot to apply the change but I think I changed this as suggested. Right, since the type generics were eliminated the generic can be on the method itself.

Done.